tasks/dataverse-handlenet.yml

```diff
@@ -0,0 +1,79 @@
+
+- name: set IndependentHandleService
+  uri:
+    url: "{{ dataverse.api.location }}/admin/settings/:IndependentHandleService"
+    method: PUT
+    body: "{{ dataverse.handlenet.independenthandleservice }}"
+    body_format: raw
+
+- name: set HandleAuthHandle
+  uri:
+    url: "{{ dataverse.api.location }}/admin/settings/:HandleAuthHandle"
+    method: PUT
+    body: "{{ dataverse.handlenet.handleauthhandle }}"
+    body_format: raw
+  when:
+   - dataverse.handlenet.handleauthhandle is defined
+
+- name: set Authority
```

Comment:
instead of `doi` should I make this `pid` in the greater role? (the Authority and Shoulder settings get repeated post-install)

Reply:
It would be more logical to call it that, although some sub-variables like `dataciterestapiurl`, `baseurl`, `mdcbaseurl`, `provider`, `username` and `password` are doi-specific. So maybe it would be logical to only move `authority`, `protocol` and `shoulder` to a new variable. I will shortly push a new commit that reflects this and also gets rid of the duplicate Authority and Shoulder setting.